static/js/config-providers.js

```diff
@@ -72,7 +72,7 @@ $(document).ready(function() { // eslint-disable-line max-lines
         var newData = [isDefault, [name, url, key, cat]];
         newznabProviders[id] = newData;
 
-        $('#editANewznabProvider').addOption(id, name);
+        $('#editANewznabProvider').prepend('<option value=' + id + '>' + name + '</option>');
         $('select#editANewznabProvider').prop('selectedIndex', 0);
```

Comment:
@Thraxis sorry, i know why, its not "adding" in the end, it adding in the top of the select. Is it possible to add in the end so `-- add new provider --` is the first one ?